### PR TITLE
Fix new zip file issue when run test with -EnableTelemetry.

### PR DESCRIPTION
**Pull request by LiliDeng**
Fix new zip file issue when run test with -EnableTelemetry. #492 
Fix issue when miss storage account which used for upload log files in secret file.

Before fix:

12/19/2018 01:40:41 : [INFO ] Creating 'C:\Users\v-lide\AppData\Local\Temp\BVT-HVMODULES-CHECK-636807516415951953.zip' from 'E:\lili\upstream\Test\TestResults\2018-1
8-12-17-33-55-4414\BVT-HVMODULES-CHECK'
12/19/2018 01:40:41 : [ERROR] Unexpected output from 7za.exe when creating E:\lili\upstream\Test\C:\Users\v-lide\AppData\Local\Temp\BVT-HVMODULES-CHECK-6368075164159
51953.zip :
12/19/2018 01:40:41 : [ERROR]  7-Zip (A) 9.20  Copyright (c) 1999-2010 Igor Pavlov  2010-11-18 Scanning  Creating archive E:\lili\upstream\Test\C:\Users\v-lide\AppDa
ta\Local\Temp\BVT-HVMODULES-CHECK-636807516415951953.zip   Error: 7-Zip cannot open file E:\lili\upstream\Test\C:\Users\v-lide\AppData\Local\Temp\BVT-HVMODULES-CHECK
-636807516415951953.zip The filename, directory name, or volume label syntax is incorrect.     System error: Unspecified error  
12/19/2018 01:40:41 : [INFO ] Getting  storage account key...
12/19/2018 01:40:44 : [ERROR] EXCEPTION : Cannot validate argument on parameter 'ResourceGroupName'. The argument is null or empty. Provide an argument that is not n
ull or empty, and then try the command again.
12/19/2018 01:40:45 : [ERROR] Source : Line 48 in script .\Utilities\UploadFilesToStorageAccount.ps1.
12/19/2018 01:40:45 : [ERROR] ERROR : / : Failed

After fix:
12/19/2018 01:43:05 : [INFO ] Creating 'C:\Users\v-lide\AppData\Local\Temp\BVT-HVMODULES-CHECK-636807517851236581.zip' from 'E:\lili\master\bugfix\Test\TestResults\2
018-18-12-17-36-40-3259\BVT-HVMODULES-CHECK'
12/19/2018 01:43:05 : [INFO ] C:\Users\v-lide\AppData\Local\Temp\BVT-HVMODULES-CHECK-636807517851236581.zip created successfully.
12/19/2018 01:43:05 : [INFO ] Using user provided storage account key.

### CODE REVIEW

File: Utilities/UploadFilesToStorageAccount.ps1 [hunk 1]
```diff
@@ -36,47 +36,37 @@ Get-ChildItem .\Libraries -Recurse | Where-Object { $_.FullName.EndsWith(".psm1"
 
 try
 {
-    if ($destinationStorageKey)
-    {
+    if ([string]::IsNullOrEmpty($destinationStorageKey)) {
         Write-LogInfo "Using user provided storage account key."
```
Comment (lizzha) [inline Utilities/UploadFilesToStorageAccount.ps1]: Update the log message?

Comment (LiliDeng) [reply Utilities/UploadFilesToStorageAccount.ps1]: Sorry I didn't catch you

Comment (LiliDeng) [reply Utilities/UploadFilesToStorageAccount.ps1]: ah, you mean I need also update the log message?

Comment (lizzha) [reply Utilities/UploadFilesToStorageAccount.ps1]: if $destinationStorageKey is empty, the script will do nothing right?

Comment (LiliDeng) [reply Utilities/UploadFilesToStorageAccount.ps1]: Previous design will still upload the zip file but throw exception when not provide key or storage account, I made a change, I will update code when key or account is empty do nothing. Let know concern.

Comment (lizzha) [reply Utilities/UploadFilesToStorageAccount.ps1]: the change is ok. just the log message should also be updated

**Pull request merged**